### PR TITLE
Add examples to make_list

### PR DESCRIPTION
**Pull request by vplauzon**

### CODE REVIEW

**Comment by PRMerger9** [comment]
@vplauzon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 1]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+    "rectangle", 4,
+    "pentagon", 5,
+    "hexagon", 6,
+    "heptagon", 7,
+    "octogon", 8,
+    "nonagon", 9,
+    "decagon", 10
+];
+shapes
+| summarize mylist=make_list(name)
```
Comment (slavikn84) [inline data-explorer/kusto/query/makelist-aggfunction.md:59]:
```suggestion
| summarize mylist = make_list(name)
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 2]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+    "rectangle", 4,
+    "pentagon", 5,
+    "hexagon", 6,
+    "heptagon", 7,
+    "octogon", 8,
+    "nonagon", 9,
+    "decagon", 10
+];
+shapes
+| summarize mylist=make_list(name) by isEvenSideCount = not(tobool(sideCount % 2))
```
Comment (slavikn84) [inline data-explorer/kusto/query/makelist-aggfunction.md:84]:
```suggestion
| summarize mylist = make_list(name) by isEvenSideCount = sideCount % 2 == 0
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 3]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+    "pentagon", 5,
+    "hexagon", 6,
+    "heptagon", 7,
+    "octogon", 8,
+    "nonagon", 9,
+    "decagon", 10
+];
+shapes
+| extend d=pack("name", name, "sideCount", sideCount)
+| summarize mylist=make_list(d) by isEvenSideCount = not(tobool(sideCount % 2))
```
Comment (slavikn84) [inline data-explorer/kusto/query/makelist-aggfunction.md:111]:
```suggestion
| summarize mylist = make_list(d) by isEvenSideCount = sideCount % 2 == 0
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 4]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+    "rectangle", 4,
+    "pentagon", 5,
+    "hexagon", 6,
+    "heptagon", 7,
+    "octogon", 8,
+    "nonagon", 9,
+    "decagon", 10
+];
+shapes
+| extend d=pack("name", name, "sideCount", sideCount)
```
Comment (slavikn84) [inline data-explorer/kusto/query/makelist-aggfunction.md:110]:
```suggestion
| extend d = pack("name", name, "sideCount", sideCount)
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 5]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
 > [!TIP]
 > Use the [`mv-apply`](./mv-applyoperator.md) operator to create an ordered list by some key. See examples [here](./mv-applyoperator.md#using-the-mv-apply-operator-to-sort-the-output-of-makelist-aggregate-by-some-key).
 
-**See also**
+## Examples
+
+### One Column
```
Comment (orspod) [inline data-explorer/kusto/query/makelist-aggfunction.md:41]:
```suggestion
### One column
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 6]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+
+Most simple example is to make a list out of a single column:
```
Comment (orspod) [inline data-explorer/kusto/query/makelist-aggfunction.md:43]:
```suggestion
The simplest example is to make a list out of a single column:
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 7]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+| summarize mylist=make_list(name)
+```
+
+|mylist|
+|---|
+|["triangle","square","rectangle","pentagon","hexagon","heptagon","octogon","nonagon","decagon"]|
+
+### Using the 'by' clause
+
+We can group by:
```
Comment (orspod) [inline data-explorer/kusto/query/makelist-aggfunction.md:68]:
```suggestion
In the following query, you group using the `by` clause:
```

File: data-explorer/kusto/query/makelist-aggfunction.md [hunk 8]
```diff
@@ -36,6 +36,86 @@ If the input to the `summarize` operator is sorted, the order of elements in the
...
+```
+
+|mylist|isEvenSideCount|
+|---|---|
+|false|["triangle","pentagon","heptagon","nonagon"]|
+|true|["square","rectangle","hexagon","octogon","decagon"]|
+
+### Packing a dynamic object
+
+We can [pack](./packfunction.md) a dynamic object in a column before making a list out of it:
```
Comment (orspod) [inline data-explorer/kusto/query/makelist-aggfunction.md:94]:
```suggestion
You can [pack](./packfunction.md) a dynamic object in a column before making a list out of it, as seen in the following query:
```

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**